Modules/Loadable/SubjectHierarchy/Widgets/qMRMLSortFilterSubjectHierarchyProxyModel.h

```diff
@@ -85,12 +85,12 @@ class Q_SLICER_MODULE_SUBJECTHIERARCHY_WIDGETS_EXPORT qMRMLSortFilterSubjectHier
   /// Overrides \sa includeNodeAttributeNamesFilter
   Q_PROPERTY(QStringList excludeNodeAttributeNamesFilter READ excludeNodeAttributeNamesFilter WRITE setExcludeNodeAttributeNamesFilter)
 
-  /// Filter to show only items that contain an attribute with this name. Empty by default
-  /// Note: Deprecated, kept only for backwards compatibility. Sets and returns the first attribute in \sa includeNodeAttributeNamesFilter
+  /// Filter to show only items that contain an item attribute with this name. Empty by default.
+  /// Note: Deprecated, kept only for backwards compatibility. Sets and returns the first attribute in \sa includeItemAttributeNamesFilter
```

Comment:
```suggestion
  /// \deprecated Kept only for backwards compatibility. Sets and returns the first attribute in \sa includeItemAttributeNamesFilter
```

Reply:
Ah, thanks! I did a search to find this (in Qt classes) but for some reason I didn't see any. I'll change it now.